packages/nx/src/command-line/import/import.ts

```diff
@@ -391,76 +392,66 @@ async function createTemporaryRemote(
-              `Add \`"workspaces": ["${pkgPath}"]\` to package.json and run "${pmc.install}".`,
+              `Added \`"workspaces": ["${pkgPath}"]\` to package.json.`,
               `See: https://yarnpkg.com/features/workspaces`,
             ]
           : pm === 'bun'
           ? [
               `We recommend enabling Bun workspaces to install dependencies for the imported project.`,
-              `Add \`"workspaces": ["${pkgPath}"]\` to package.json and run "${pmc.install}".`,
+              `Added \`"workspaces": ["${pkgPath}"]\` to package.json.`,
               `See: https://bun.sh/docs/install/workspaces`,
             ]
           : [
               `We recommend enabling PNPM workspaces to install dependencies for the imported project.`,
-              `Add the following entry to to pnpm-workspace.yaml and run "${pmc.install}":`,
+              `Added the following entry to to pnpm-workspace.yaml:`,
               chalk.bold(`packages:\n  - '${pkgPath}'`),
               `See: https://pnpm.io/workspaces`,
             ],
     });
   } else {
-    // Check if the new package is included in existing workspaces entries. If not, warn the user.
-    let workspaces: string[] | null = null;
-
-    if (pm === 'npm' || pm === 'yarn' || pm === 'bun') {
-      const packageJson = readPackageJson();
-      workspaces = packageJson.workspaces;
-    } else if (pm === 'pnpm') {
-      const yamlPath = join(workspaceRoot, 'pnpm-workspace.yaml');
-      if (existsSync(yamlPath)) {
-        const yamlContent = await fsp.readFile(yamlPath, 'utf-8');
-        const yaml = yamlLoad(yamlContent);
-        workspaces = yaml.packages;
-      }
-    }
-
-    if (workspaces) {
-      const isPkgIncluded = workspaces.some((w) => minimatch(pkgPath, w));
-      if (!isPkgIncluded) {
-        const pkgsDir = dirname(pkgPath);
-        output.warn({
-          title: `Project missing in workspaces`,
-          bodyLines:
-            pm === 'npm' || pm === 'yarn' || pm === 'bun'
-              ? [
-                  `The imported project (${pkgPath}) is missing the "workspaces" field in package.json.`,
-                  `Add "${pkgsDir}/*" to workspaces run "${pmc.install}".`,
-                ]
-              : [
-                  `The imported project (${pkgPath}) is missing the "packages" field in pnpm-workspaces.yaml.`,
-                  `Add "${pkgsDir}/*" to packages run "${pmc.install}".`,
-                ],
-        });
-      }
-    }
+    output.warn({
+      title: `Project missing in workspaces`,
+      bodyLines:
+        pm === 'npm' || pm === 'yarn' || pm === 'bun'
+          ? [
+              `The imported project (${pkgPath}) is missing the "workspaces" field in package.json.`,
+              `Added "${pkgPath}" to workspaces.`,
+            ]
+          : [
+              `The imported project (${pkgPath}) is missing the "packages" field in pnpm-workspaces.yaml.`,
+              `Added "${pkgPath}" to packages.`,
+            ],
```

Comment:
Can we make this a success message instead of a warning? It should tell people that nx has successfully added the path to the workspaces/packages field.

Reply:
changed